using-mobile-extensions/adobe-campaign-standard/README.md

```diff
@@ -310,10 +334,236 @@ iOS simulators do not support push messaging.
 {% endhint %}
 
 To obtain the registration ID/token, see [Configuring Remote Notification Support](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/HandlingRemoteNotifications.html#//apple_ref/doc/uid/TP40008194-CH6-SW1).
+
+#### Objective-C
+
+#### Example
+
+```objectivec
+- (void) application:(UIApplication *)application didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken {
+  // Set the deviceToken that the APNS has assigned to the device
+  [ACPCore setPushIdentifier:deviceToken];
+  //...
+}
+```
+
+#### Swift
+
+#### Example
+
+```swift
+func application(_ application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
+  // Set the deviceToken that the APNS has assigned to the device
+  ACPCore.setPushIdentifier(deviceToken)
+  //...
+}
+```
+
 {% endtab %}
 
 {% tab title="React Native" %}
+
 Follow instructions in the Android/iOS tabs to set up platform-specific push configuration and use the following API in your React Native project:
```

Comment:
Please reword as follows:
"To set up platform-specific push configuration, and use the following API in your React Native project, complete the steps in the **Android** and **iOS** tabs.